test/onnx/test_fx_to_onnx_with_onnxruntime.py

```diff
@@ -464,25 +466,6 @@ def forward(self, tensor_x: torch.Tensor):
             self, MNISTModel(), (tensor_x,)
         )
 
-    @pytorch_test_common.skip_min_ort_version(
-        reason="ORT doesn't support dynamic fx exporter yet making SegFault flaky test",
-        version="1.15",
-        dynamic_only=True,
-    )
-    # test single op with no kwargs
-    def test_sigmoid(self):
```

Comment:
is this expected?

Reply:
Yes, we have `test_sigmoid_add` already. I recall this one was like step-by-step test when we were in early stage of developing atenlib. Besides, we will land fx exporter version of test_ops_consistency.py anyway.